stm32-pbv4.ld

```diff
@@ -115,11 +115,6 @@ SECTIONS
 	/DISCARD/ : { *(.eh_frame) }
 
 	. = ALIGN(4);
-
-	 .text : {
-		FILL(0)
-		. = (0x8000 - _edata + _data);
-	} >rom
```

Comment:
Knowing nothing about this particular MCU's memory layout - it looks like this is supposed to be located at the end of `.text` proper and before `.data`?

Worth checking we don't run into [this issue](https://github.com/srobo/power-v4-fw/commit/dd61a24857b6b186017126cddb0479c1445a82fd) again by removing this.

Reply:
That's a valid point, the tools we're intending to use do have an option to erase the rest of flash but, since the binary would be truncated, the CRC may still be wrong. I'll test this once I have a kit but ulitmately I can't see how this previously worked.

Sounds like I'm due another deep dive into linkerscripts.